kintone/cli-kintone
# docs: define the Stability Index

## Why



cli-kintone follows the semantic versioning.
However, we will provide some unstable features that are under development.
Also, some features might be deprecated.

In this PR, we define the stability index inspired by [of Node.js](https://nodejs.org/docs/latest-v18.x/api/documentation.html#stability-index) to indicate which feature is experimental or deprecated.

## What



- add/update document
  - add the stability index page
  - update contributing page
  - update versioning policy, adding a link to stability index

## How to test



See the following previews:

- https://docs-define-stability-index.d28lu13lljyj0a.amplifyapp.com/community/stability-index
- https://docs-define-stability-index.d28lu13lljyj0a.amplifyapp.com/community/contributing/#stability
- https://docs-define-stability-index.d28lu13lljyj0a.amplifyapp.com/community/versioning/

Or run `pnpm doc:start` to check locally.

## Checklist
